packages/wrangler/src/dev.tsx

```diff
@@ -585,7 +585,7 @@ export async function startDev(args: StartDevOptions) {
 				});
 			}
 
-			if (process.stdout.isTTY && (args.showInteractiveDevSession ?? true)) {
+			if (process.stdout.isTTY && args.showInteractiveDevSession !== false) {
```

Comment:
This should use `isNonInteractiveOrCI()`

Reply:
https://github.com/cloudflare/workers-sdk/pull/6149/commits/a88b702c2423ab88a679a598dc59a9ce19cc5d09